glyphs-reader/src/font.rs

```diff
@@ -588,6 +627,30 @@ impl Font {
     }
 }
 
+impl Font {
+    pub fn default_master_idx(&self) -> usize {
+        // https://github.com/googlefonts/fontmake-rs/issues/44
+        custom_param(&self.other_stuff, "Variable Font Origin")
+            .map(|(_, param)| match param {
+                Plist::Dictionary(dict) => dict.get("value"),
+                _ => None,
+            })
+            .map(|value| {
+                let Some(Plist::String(origin)) = value else {
+                    warn!("Incomprehensible Variable Font Origin");
+                    return 0;
+                };
+                self.font_master
+                    .iter()
+                    .enumerate()
+                    .find(|(_, master)| &master.id == origin)
+                    .map(|(idx, _)| idx)
+                    .unwrap_or_default()
+            })
+            .unwrap_or_default()
```

Comment:
took me a bit to understand this, i'm not a big fan of these complicated chains of method calls, but if this is supposed to be idiomatic rust, oh well...

Reply:
I think you'll find it grows on you. THe code reads left to right, top to bottom like language in a way thats entirely not true of the normal way of doing it.

Reply:
To be fair, it's also very possible I still write un-idiomatic Rust

Reply:
I think I made it somewhat clearer.